lib/build-view.js

```diff
@@ -95,10 +94,6 @@ module.exports = (function() {
           this.detach();
         }
         break;
-
```

Comment:
It's not really necessary to open the panel when the config changes. Also, config subscriptions are triggered on package activation, so anything you do here also happens every time an Atom window is opened (not just when the value changes). So it's better to not do too much here.